ipymol/core.py

```diff
@@ -64,11 +65,10 @@ def display(self):
         fig = plt.figure(figsize=(20, 20))
         ax = fig.add_subplot(111)
         ax.axis('off')
-        fh = tempfile.NamedTemporaryFile()
-        self._server.do("cmd.png('%s')" % fh.name)
+        fh = tempfile.NamedTemporaryFile(suffix='.png')
+        self._server.do("png {};".format(fh.name))
         ax.imshow(mpimg.imread(fh.name))
-        os.close(fh)
-        return fig
+        #return fig
```

Comment:
why is this commented out? It can helpful to have the ability to modify the figure outside of `display`

```suggestion
        return fig
```

Reply:
When I used ipymol in jupyter notebook, image is displayed two times. I thought it is due to I used %matplotlib inline option, but I would like to draw image only one time.
I think image is drawn without return fig if user uses %matplotlib inline option. Any comments and suggestions are highly appreciated.
Thanks,
Taka

Reply:
This is because of a [quirk in matplotlib figures](https://stackoverflow.com/a/35423325). You can either catch (`_ = viewer.display()`) or suppress (`viewer.display();`) the output figure to avoid this behavior.